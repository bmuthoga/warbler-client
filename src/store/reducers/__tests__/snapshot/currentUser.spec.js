import authReducer from '../../currentUser'
import { SET_CURRENT_USER } from '../../../actionTypes'

describe('Auth Reducer Snapshot Tests', () => {
  it('should return the initial state', () => {
    expect(authReducer(undefined, {})).toMatchSnapshot()
  })

  it('should handle SET_CURRENT_USER', () => {
    const user = {
      id: '89498gh94ug95g0',
      username: 'testUser',
      profileImageUrl: 'http://lorempixel.com/200/200/'
    }
    expect(authReducer(undefined, { type: SET_CURRENT_USER, user })).toMatchSnapshot()
  })
})
