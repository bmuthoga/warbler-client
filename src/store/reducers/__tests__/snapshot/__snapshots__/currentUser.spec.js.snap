// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Reducer Snapshot Tests should handle SET_CURRENT_USER 1`] = `
Object {
  "isAuthenticated": true,
  "user": Object {
    "id": "89498gh94ug95g0",
    "profileImageUrl": "http://lorempixel.com/200/200/",
    "username": "testUser",
  },
}
`;

exports[`Auth Reducer Snapshot Tests should return the initial state 1`] = `
Object {
  "isAuthenticated": false,
  "user": Object {},
}
`;
